gallery/visualtests/nxDrawer.js

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { setupBrowser } = require('./testUtils');
+
+describe('NxDrawer', function() {
+  const {
+    waitAndGetElements,
+    checkScreenshot,
+    getPage,
+    a11yTest,
+    wait
+  } = setupBrowser('#/pages/Drawer');
+
+  async function openDrawer(drawerId, buttonId) {
+    const openDrawerBtnSelector = `#${buttonId}`;
+
+    const [openDrawerBtn] = await waitAndGetElements(openDrawerBtnSelector);
+
+    await openDrawerBtn.click();
+
+    await wait(200);
```

Comment:
`waitAndGetElements` below should already wait up to 30 seconds, so I doubt this is necessary

Reply:
> `waitAndGetElements` below should already wait up to 30 seconds, so I doubt this is necessary

Oh we added that to wait for the transition to complete before taking the screenshot but it should probably be moved to before the screenshot part of the test. 👍

Reply:
> Oh we added that to wait for the transition to complete before taking the screenshot but it should probably be moved to before the screenshot part of the test. +1

There's lots of `wait(200)`s now.  Could make a `waitForAnimation` helper function local to this file to encapsulate that.

Reply:
> > Oh we added that to wait for the transition to complete before taking the screenshot but it should probably be moved to before the screenshot part of the test. +1
> 
> There's lots of `wait(200)`s now. Could make a `waitForAnimation` helper function local to this file to encapsulate that.

Good call, I've updated it here: e7ba944ff5d97234028db574826f3e7ab0fe161c